game/db.py

```diff
@@ -973,99 +912,7 @@
 Units separated by country. 
 country : DCS Country name
 """
-FACTIONS: Dict[str, Dict[str, Any]] = {
-
-    "Bluefor Modern": BLUEFOR_MODERN,
-    "Bluefor Cold War 1970s": BLUEFOR_COLDWAR,
-    "Bluefor Cold War (with A-4)": BLUEFOR_COLDWAR_A4,
-    "Bluefor Cold War (with A-4 and MB339)": BLUEFOR_COLDWAR_MODS,
-
-    "USA 1955 (WW2 Pack)": USA_1955,
-    "USA 1960": USA_1960,
-    "USA 1965": USA_1965,
-    "USA 1990": USA_1990,
-    "USA 2005": USA_2005,
-    "USA Aggressors 2005": US_Aggressors,
-
-    "Russia 1955": Russia_1955,
-    "Russia 1965": Russia_1965,
-    "Russia 1975": Russia_1975,
-    "Russia 1990": Russia_1990,
-    "Russia 2010": Russia_2010,
-    "Russia 2020 (Modded)": Russia_2020,
-
-    "France 1995": France_1995,
-    "France 2005": France_2005,
-    "France 2005 (Modded)": France_2005_Modded,
-
-    "Germany 1990": Germany_1990,
-
-    "Netherlands 1990": Netherlands_1990,
-
-    "United Kingdom 1990": UnitedKingdom_1990,
-
-    "Spain 1990": Spain_1990,
-
-    "Italy 1990": Italy_1990,
-    "Italy 1990 (With MB339)": Italy_1990_MB339,
-
-    "Israel 2000": Israel_2000,
-    "Israel 1982": Israel_1982,
-    "Israel 1973 (WW2 Pack)": Israel_1973,
-    "Israel 1973": Israel_1973_NO_WW2_UNITS,
-    "Israel 1948": Israel_1948,
-
-    "Arab Armies 1982": Syria_1982,
-    "Arab Armies 1973": Syria_1973,
-    "Arab Armies 1967 (WW2 Pack)": Syria_1967_WW2_Weapons,
-    "Arab Armies 1967": Syria_1967,
-    "Arab League 1948": Arab_Armies_1948,
-
-    "China 2010": China_2010,
-
-    "Sweden 1990": Sweden_1990,
-
-    "Australia 2005": Australia_2005,
-
-    "Canada 2005": Canada_2005,
-
-    "Japan 2005": Japan_2005,
-
-    "Turkey 2005": Turkey_2005,
-
-    "United Arab Emirates 2005": UAE_2005,
-
-    "Ukraine 2010": Ukraine_2010,
-
-    "India 2010": India_2010,
-
-    "Libya 2011": Libya_2011,
-
-    "Syria 2011": Syria_2011,
-
-
-    "Pakistan 2015": Pakistan_2015,
-
-    "Iran 2015": Iran_2015,
-
-    "North Korea 2000": NorthKorea_2000,
-
-    "Insurgent": Insurgent,
-    "Insurgent (Modded)": Insurgent_modded,
-
-    "PMC (American)": PMC_WESTERN_A,
-    "PMC (American) - MB339": PMC_WESTERN_B,
-    "PMC (Russian)": PMC_RUSSIAN,
-
-    "Allies 1944 (WW2 Pack)": USA_1944,
-    "USA 1944 (WW2 Pack)": ALLIES_1944,
-    "UK 1944 (WW2 Pack)": UK_1944,
-
-    "Germany 1944 (WW2 Pack)": Germany_1944,
-    "Germany 1944 Easy (WW2 Pack)": Germany_1944_Easy,
-
-}
-
+FACTIONS: [Faction] = faction_loader.load_factions()
```

Comment:
Convert this to a class with a `@classmethod` like I did for the campaigns. Running code that might log during initialization can break logging.